src/translib/transformer/xfmr_ztp.go

```diff
@@ -342,7 +342,7 @@ var DbToYang_ztp_config_xfmr SubTreeXfmrDbToYang = func (inParams XfmrParams) (e
 
 var YangToDb_ztp_config_xfmr SubTreeXfmrYangToDb = func(inParams XfmrParams) (map[string]map[string]db.Value,error) {
     var err error
-    log.Info("TableXfmrFunc - Uri ZTP: ", inParams.uri);
+    xfmrLogInfoAll("TableXfmrFunc - Uri ZTP: %v", inParams.uri);
     pathInfo := NewPathInfo(inParams.uri)
     targetUriPath, err := getYangPathFromUri(pathInfo.Path)
     log.Info("TARGET URI PATH ZTP:", targetUriPath)
```

Comment:
We initially made changes to the logs where parameters were printed from inParams. But doing this only when we print dbDataMap in inParams only as this can contains huge data.